<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProjectWindow</class>
 <widget class="QMainWindow" name="CreateProjectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New project</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Project Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="prNameEdit">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>361</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="projPersp2D">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>102</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>2D</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="projPersp3D">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>90</y>
      <width>102</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>3D</string>
    </property>
   </widget>
   <widget class="QPushButton" name="prCreateBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>400</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dirBrouseBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>140</y>
      <width>80</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>151</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Root directory path</string>
    </property>
   </widget>
   <widget class="QLabel" name="rootPathField">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>140</y>
      <width>58</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>(none)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
